Extension/src/LanguageServer/client.ts

```diff
@@ -2187,6 +2197,12 @@ export class DefaultClient implements Client {
         await this.languageClient.sendNotification(DidOpenNotification, params);
     }
 
+    public async getChatContext(): Promise<ChatContextResult | undefined> {
+        await this.ready;
+        const result: ChatContextResult | undefined = await this.languageClient.sendRequest(CppContextRequest, null);
```

Comment:
Is there a reason for the result variable versus just returning it?

Reply:
removed the local var.